### PR TITLE
*: bump 0.7.1

### PR DESCRIPTION
**Pull request by hunterlxt**
Signed-off-by: Xintao <hunterlxt@live.com>

### CODE REVIEW

**Comment by hunterlxt** [comment]
@BusyJay @yiwu-arbug PTAL

**Review by yiwu-arbug — APPROVED** [review]
LGTM

File: proto/Cargo.toml [hunk 1]
```diff
@@ -19,7 +19,7 @@ prost-codec = ["prost-derive", "bytes", "lazy_static", "grpcio/prost-codec", "pr
 
 [dependencies]
 futures = "0.3"
-grpcio = { path = "..", features = ["secure"], version = "0.7.0", default-features = false }
+grpcio = { path = "..", features = ["secure"], version = "0.7.1", default-features = false }
```
Comment (BusyJay) [inline proto/Cargo.toml:22]: Can you also change the version of grpcio-proto to 0.7? I forgot to do so last time.

**Review by BusyJay — COMMENTED** [review]
Rest LGTM

File: proto/Cargo.toml (hunk 1 above)
Comment (hunterlxt) [reply proto/Cargo.toml:22]: Done

**Review by BusyJay — APPROVED** [review]

**Review by yiwu-arbug — APPROVED** [review]
LGTM

**Pull request merged**